Mathlib/RingTheory/Ideal/Quotient/Basic.lean

```diff
@@ -138,6 +138,7 @@ theorem maximal_of_isField (I : Ideal R) (hqf : IsField (R ⧸ I)) : I.IsMaximal
     exact J.sub_mem (J.mul_mem_right _ hxJ) (hIJ (Ideal.Quotient.eq.1 hy))
 
 /-- The quotient of a ring by an ideal is a field iff the ideal is maximal. -/
+@[stacks 09FF]
```

Comment:
I don't think this tag is right.

Reply:
Again, this corresponds to the first paragraph 

To get started, let us begin by providing several examples of fields. The reader should recall that if R is a ring and I⊂R an ideal, then R/I is a field precisely when I is a maximal ideal.

in section 09FF. This tag is for the whole section. I'll delete this tag.